### ISSUE TITLE
Chrome Library Detector causes faulty Back behavior

### ISSUE THREAD

**Issue by hced** [ttsvetko/Chrome-Library-Detector-Extension#2]
I've have an issue that I have ruled out to be caused by Chrome Library Detector:

The Back/Forward history nagivation doesn't work properly. Here's a use case to replicate the problem:
1. Make a Google search, e.g: "sample wikipedia"
2. Click the Wikipedia article result (should be first).
3. Click the topmost link on the Wikipedia page; "Sample (Statistics)".
4. Hit the browser's Back button (or corresponding keyboard shortcut).
5. You end up on Google's page again! Whereas you _should_ go back to the first Wikipedia page you visited.

In other words: in my case, this extension makes me _leave the domain_ no matter how many pages I've visited within the domain.

**Comment by ttsvetko** [comment]
Hi, Yeha, thats an odd issue. I'll do a fix and will upload it, meanwhile i'm reverting back to prev version to avoid such browsing issues that are really unpleasant.

**Comment by ttsvetko** [comment]
It seems to be a chrome issue (http://code.google.com/p/chromium/issues/detail?id=68805&q=location%20javascript%20void&colspec=ID%20Pri%20Mstone%20ReleaseBlock%20Area%20Feature%20Status%20Owner%20Summary). Will try to work-around it.

**Comment by hced** [comment]
Thanks for looking it up, ttsvetko! FYI I'm using Chrome stable, version 17.0.963.83. Haven't checked dev. version or Canary but assume it's the same there.

**Comment by ttsvetko** [comment]
I make assumption where should be the issue, but this weekend will see it in details. The code that test pages for libraries is the same, the difference is in when page should be tested... I'will update the issue here when it is fixed. Thanks again for spotting this.

**Comment by hced** [comment]
You rock, ttsvetko. No hurry. Glad to help.
